spec/unit/blockscore/person_spec.rb

```diff
@@ -14,5 +38,12 @@ module BlockScore
       expect(person.invalid?).to eq(true)
       expect(person.valid?).to eq(false)
     end
+
+    it "loaded" do
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.